hosting/hooks/use-user-session.ts

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useState } from "react";
+
+import { onAuthStateChanged } from "../libs/firebase/auth";
```

Comment:
It would be better if we use a path alias which will make imports cleaner and more concise